### PR TITLE
test: Support single-source sync

### PR DESCRIPTION
**Pull request by jannden**
Closes: https://github.com/famedly/famedly-sync/issues/81

This fixes tests for: https://github.com/famedly/famedly-sync/pull/79

As per this [comment](https://github.com/famedly/famedly-sync/pull/79#issuecomment-2449942466).

### CODE REVIEW

**Comment by jannden** [comment]
As mentioned in the original PR [in this comment](https://github.com/famedly/famedly-sync/pull/79#issuecomment-2453335474), we first need to clarify how to approach the clash between LDAP and CSV sync. Then a small adjustment to the `test_e2e_full_sync` is expected and this should be ready to go.

**Review by emgrav — APPROVED** [review]

File: config.sample.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ feature_flags:
   # - deactivate_only # Only deactivate users, do not create or update them. Keep in mind LDAP is cached and all the changes made on LDAP will be written to the cache as if they where applied. Therefore, only the deactivation changes will be applied to Zitadel but **all the other changes will be lost**.
 
 # Configuration for the sources to sync from.
+# IMPORTANT: Only one source can be present at a time.
```
Comment (tlater-famedly) [inline config.sample.yaml:22]: I think the sample config should be split in three for this, don't want to document something that can never work.

**Review by tlater-famedly — COMMENTED** [review]
Looks good, I like this!

File: config.sample.yaml (hunk 1 above)
Comment (jannden) [reply config.sample.yaml:22]: Okay, let's split the sample as part of the documentation commit on the feature branch.

**Pull request merged**